### DISCUSSION TITLE
一些xdisp_virt的演示视频.

### DISCUSSION THREAD

**Discussion by fanxiushu** [fanxiushu/xdisp_virt#14] [General]
一些xdisp_virt的演示视频.

https://user-images.githubusercontent.com/19388676/188848937-71ee7f4d-e8fe-4928-90e6-998dfc4264e9.mp4

**Comment by N0exp** [comment]
大佬，请教一下，哪个微软的间接显示器驱动如何调用IddCXMonitorSetupHardwareCursor解决抓图中有鼠标的问题？我调用始终返回0xc000000d参数错误，请问大佬是如何解决这个问题的？求大佬教教
